scipy/ndimage/_measurements.py

```diff
@@ -307,6 +307,91 @@ def find_objects(input, max_label=0):
     return _nd_image.find_objects(input, max_label)
 
 
+def value_indices(arr, *, nullval=None):
+    """
+    Find indices of each distinct value in given array.
+
+    Parameters
+    ----------
+    arr : ndarray of ints
+        Array containing integer values.
+    nullval : int, optional
+        This value will be ignored in searching the arr array. If not
+        given, all values found will be included in output.
```

Comment:
Would `ignore_value` be a better name for this parameter? The term `null` is common in some image processing contexts, but certainly not all, and maybe this would be clearer?

Reply:
I think that would be better yes.